js/src/lib/util.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import fetch from 'unfetch';
```

Comment:
Oh cool, 500 bytes is fine! I thought it was going to be large, in which case I would have suggested using XMLHttpRequest directly 😛